plugins/doc_fragments/foreman.py

```diff
@@ -276,3 +276,19 @@ class ModuleDocFragment(object):
     required: true
     type: str
 '''
+
+    DATASTREAM = '''
```

Comment:
Should this be named `SCAP_DATASTREAM` as it is rather specific?

Reply:
yes

Reply:
I've also renamed the helper class to match